style.css

```diff
@@ -1168,6 +2634,98 @@ body {
     height: 24px;
     margin-top: 16px;
   }
+
+  .second-page__article {
+    width: 332px;
+    padding: 18px 36px 30px 41px;
+    margin-bottom: 16%;
+  }
+
+  .second-page__title {
+    margin-bottom: 2px;
+    font-size: 26px;
+    letter-spacing: -.5px;
+  }
+
+  .second-page__subtitle {
+    font-size: 13px;
+    margin-bottom: 18px;
+  }
+
+  .second-page__description {
+    width: 315px;
+    margin-bottom: 26px;
+    font: 600 12px/16px 'Montserrat-Regular', sans-serif;
+    letter-spacing: -.3px;
+  }
+
+  .second-page__color-options {
+    margin-bottom: 45px;
+  }
+
+  .eclipse {
+    width: 30px;
+    height: 30px;
+    margin-right: 16px;
+  }
+
+  .second-page__price {
+    margin-right: 9px;
+    margin-bottom: 43px;
+    font-size: 26px;
+  }
+
+  .second__add-to-cart {
+    font-size: 17px;
+    padding: 12px;
+  }
+
+  .third-page {
+    background: linear-gradient(to right, #EBE3D8 131px, #000 131px, #000 183px, #EBE3D8 183px);
+  }
+
+  .third-page__wrap {
+    width: 58%;
+  }
+
+  .third-page__article {
+    padding: 29px 25px 35px 30px;
+    margin-bottom: 5px;
+    width: 120%;
+  }
+
+  .third-page__title {
+    font-size: 26px;
+    letter-spacing: -.2px;
+  }
+
+  .third-page__subtitle {
+    margin-bottom: 9px; 
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)